rvv-intrinsic-generator/rvv_intrinsic_gen/generator.py

```diff
@@ -128,6 +128,84 @@ def is_support_overloaded(name, **kwargs):
 
     return True
 
+
+  @staticmethod
+  def adjust_gnu_api_count(api_count, test_file, has_policy):
+    if test_file == "vmv.C":
+      api_count = 178 if has_policy else 110
```

Comment:
Magic number here seems a bit strange. Maybe add some comments here for more context.